### ISSUE TITLE
Retirement of SocialSciences task view

### ISSUE THREAD

**Issue by zeileis** [cran-task-views/ctv#8]
The existing [Social Sciences task view](https://CRAN.R-project.org/view=SocialSciences) is one of the old existing task views that was useful when it was introduced but turned out to be too broad to be useful. John Fox @john-d-fox and I both agree on this and it would probably be better to establish separate task views for topics like: mixed-effects models, social network analysis, demography, and possibly causal analysis or matching/propensity scores. Unless there are any objects I therefore suggest:

* We retire the SocialSciences task view immediately (still in the SVN on R-Forge).
* When we announce the possibility of proposing new task views, we reach out to some people who might be willing to put together task views on the topics above.

**Comment by tuxette** [comment]
I agree with a few comments:
- I suppose that you are aware that some people have already been working on a task view on causal analysis, which is (as far as I know) almost ready to be submitted?
- I wonder if the task view "social network analysis" is not a bit overlapping "graphical models"?

**Comment by zeileis** [comment]
Thanks, Nathalie @tuxette !

_Causal inference:_ Julie Josse once mentioned to me that she might be interested in putting together a task view on that topic but that was 2.5 years ago and I hadn't heard back since.

_Social network analysis:_ I'm not sure how much overlap there really is but I'll mention it to Søren in a separate issue.

**Comment by john-d-fox** [comment]
HI,
 
I can’t speak to existing plans for a task view on causal inference (thought that would be good news), but I did take a look at the graphical models task view, and there is overlap with network analysis. Some of the overlap makes sense,
 such as packages for drawing graphs, but I find some of it a bit mystifying, such as the inclusion of the ergm package (for exponential random graph models), which, unless I’m missing something, is unrelated to graphical models. I think that there’s still
 probably room for a task view that focuses on social network analysis, possibly coordinating with the content of the graphical models task view.
 
Best,
John
 

From:
Achim Zeileis ***@***.***>
Reply-To: cran-task-views/ctv ***@***.***>
Date: Tuesday, December 28, 2021 at 7:49 AM
To: cran-task-views/ctv ***@***.***>
Cc: John Fox ***@***.***>, Mention ***@***.***>
Subject: Re: [cran-task-views/ctv] Retirement of SocialSciences task view (Issue #8)


 

Thanks, Nathalie 
@tuxette !
Causal inference: Julie Josse once mentioned to me that she might be interested in putting together a task view on that topic but that was 2.5 years ago and I hadn't
 heard back since.
Social network analysis: I'm not sure how much overlap there really is but I'll mention it to Søren in a separate issue.
—
Reply to this email directly, 
view it on GitHub, or 
unsubscribe.
Triage notifications on the go with GitHub Mobile for 
iOS or 
Android. 
You are receiving this because you were mentioned.Message
 ID: ***@***.***>

**Comment by zeileis** [comment]
I personally also lean more towards a separate task view but wanted to get Søren's opinion on this.

**Comment by john-d-fox** [comment]
I agree that makes sense, and it would be desirable to coordinate the content of the two task views if a social networks task view materializes.
 

From:
Achim Zeileis ***@***.***>
Reply-To: cran-task-views/ctv ***@***.***>
Date: Tuesday, December 28, 2021 at 11:19 AM
To: cran-task-views/ctv ***@***.***>
Cc: John Fox ***@***.***>, Mention ***@***.***>
Subject: Re: [cran-task-views/ctv] Retirement of SocialSciences task view (Issue #8)


 

I personally also lean more towards a separate task view but wanted to get Søren's opinion on this.
—
Reply to this email directly, 
view it on GitHub, or 
unsubscribe.
Triage notifications on the go with GitHub Mobile for 
iOS or 
Android. 
You are receiving this because you were mentioned.Message
 ID: ***@***.***>

**Comment by eddelbuettel** [comment]
@john-d-fox You email setup makes replies borderline readable.  Can you consider another email client, or answers here?  Editing your second answer down to something more useful.

**Comment by john-d-fox** [comment]
Dirk,

Until recently, I used the Thunderbird email client, which was easily configurable, but my university IT department now requires use of either the Outlook or Apple Mail clients, or Outlook web mail. I'm using the Outlook email client, and have found it frustrating.

OTOH, I just looked at my messages in this thread on GitHub, and, apart from a bad line break and a typo in my first message, they seem perfectly intelligible to me.

My second message concerns coordinating a new social networks task view, should one materialize, with the existing graphical models task view. That, of course, would be up to the maintainers of the two task views. Is that sufficiently clear?

John

**Comment by eddelbuettel** [comment]
Well yes, after I edited them.  Mail replies often leave the quoted trail, and GitHub is clever enough to not show that for the forms it recognized.  But that failed here.  But what was a first I had never seen in all those years is that the mailer you used (possibly in the way IT configured it) also spammed in thirty or so lines of CSS.   Which is fine and well _if_ done in a way that GitHub can filter it which it didn't here.  It's quick enough to edit out as a one-off (via the the '...' menu entry) but can get repetitive.  The much better news is that this reply of your has neither shortcoming.  So thank you!

**Comment by john-d-fox** [comment]
Now I understand -- the message was garbled before you edited it. Thanks for that (and I've now fixed the typo and bad line break).

I managed a while ago to configure Outlook so that it sends plain-text emails by default, but I can't get it to reply in plain text to HTML emails. As I said in my previous message, I wish that I could use a better email client, but I can't.

I'll simply continue to avoid using email to reply to GitHub messages.

**Comment by robjhyndman** [comment]
Maybe @timriffe would be interested in doing a Demography task view?

**Comment by timriffe** [comment]
@robjhyndman sure but I couldn't dedicate much time until May/June. But it wouldn't be too hard to rally support, would it @MJAlexander ?

**Comment by zeileis** [comment]
Thanks for the suggestion Rob @robjhyndman and for the follow-up Tim @timriffe , this is very much appreciated. Time is not crucial here. If you and a couple of potential co-maintainers like @MJAlexander would put together a first proposal, then we (the CRAN Task View Editors) can have a look and you will still have enough time to come up with the actual first version of the full task view. The guidelines for proposing a new task view are described at: <https://github.com/cran-task-views/ctv/blob/main/Proposal.md>. Let us know if you need more information or feedback etc.

**Comment by zeileis** [comment]
OK, thanks everyone for the discussion so far. I will close this issue now and just note some ideas and concrete plans:

* A `CausalInference` task view has already been proposed in https://github.com/cran-task-views/ctv/issues/15
* A `Demography` task view might be proposed by @timriffe and potentially @MJAlexander
* A `SocialNetworkAnalysis` task view would also be a good idea but no potential maintainers have been encouraged to submit a proposal.

Further "spin-offs" might be considered in the future as well.

**Comment by john-d-fox** [comment]
One further suggestion: The people behind statnet 
<https://statnet.org/attribution> (I don't know them) might be 
interested in a social-networks taskview.

**Comment by zeileis** [comment]
Thanks, I agree. They would be a natural group to ask.

**Comment by john-d-fox** [comment]
Yet another suggestion: I originally mentioned a mixed-effects models taskview as one possible partial successor to the social-sciences taskview. A natural person to approach about that would be Ben Bolker. I do know him and could approach him, if that would be helpful. (And is someone planning to follow up with the statnet folks about a social networks taskview?)

**Comment by zeileis** [comment]
We're currently working towards the first official release of the task views based on the new infrastructure. So I would like to get this out first before starting to invite new task view proposals.

Also, in case of the mixed-effects topic I'm not completely sure that this would be the best way to encapsulate a new topic. There is a lot of overlap with existing task views and it would exclude non-mixed-effects methods for the same kind of data (longitudinal, panel, multi-level), e.g., OLS variants or GEE.
